xcode/Subconscious/Shared/Components/Common/Story/StoryUserView.swift

```diff
@@ -105,7 +106,23 @@ struct StoryUserView: View {
                                 }
                             )
                         }
-                       
+                        
+                        switch story.user.resolutionStatus {
+                        case .unresolved:
+                            Button(
+                                action: {
+                                    onRefreshUser()
+                                },
+                                label: {
+                                    Label(
+                                        title: { Text("Refresh") },
```

Comment:
My gut is that an explicit refresh control feels like exposing too much implementation detail to the user.

Could it be possible to invoke this automatically for any unresolved profile, or tie it into following list pull-to-refresh?

Non-blocking. We can file a follow-up to address.

Reply:
Tying it to pull-to-refresh is a good idea, I'll experiment with that before merging to investigate.

Reply:
As discussed, attempting resolution again on-tap feels way better.


https://github.com/subconsciousnetwork/subconscious/assets/5009316/f4e69a8d-d6f5-4e28-8a3e-b9dea34884cd